docs/local-module.md

```diff
@@ -0,0 +1,46 @@
+# Postman.gov.sg
+
+- [Background](#background)
+- [How to install](#how-to-install)
+- [Local development](#local-development)
+- [Build dependencies](#build-dependencies)
+- [More information](#more-information)
+
+## Background
+
+We wanted to find a way to share the templating logic because:
+
+1. Used in frontend, backend & worker.
+2. Better consistency, we just have change the code in one place rather than three.
+
+There are several options that we considered:
+
+- npm install from separate repo (postmangovsg-js-tools?)
+- npm pack and copy tarball into respective folders
+- Publish to npm registry
+- Separate branch (sdk branch)
+- Use lerna to manage multi-package repo
+- local node module
+
+We decided to use local shared module as the rest of the options are more complicated or requires alot more time & effort to set them up.
+
+## How to install
+
+Any shared module can be installed using `npm install ../modules/postman-templating`.
+
+## Local development
+
+To test the changes you want to make to a module like `postman-templating`:
+
+1. Make changes to the module.
+2. Run `npm start build` in `postman-templating`.
+
+`npm start build` will remove & replace the `build` folder in `postman-templating` with the updated change.
+
```

Comment:
This part is technically correct, but just doesn't paint the full picture. 
Eg. If I were to be testing my changes made to `postman-templating` in the `/frontend` directory, I will also have to run `npm install` or `npm ci` **under the `frontend` directory** for the new `postman-templating` module to be built there.

Reply:
Hmmm i tested on the backend and it seems like i didn't have to `npm install` or `npm ci`, let me verify it.